src/image/ImageAnnotator.ts

```diff
@@ -112,8 +116,16 @@ export default class ImageAnnotator extends BaseAnnotator {
             return;
         }
 
+        let shape = null;
         if (isRegion(annotation)) {
-            const transformedShape = getTransformedShape(annotation.target.shape, rotation);
+            // eslint-disable-next-line prefer-destructuring
+            shape = annotation.target.shape;
+        } else if (isDrawing(annotation)) {
+            shape = getShape(annotation.target.path_groups);
+        }
+
+        if (shape) {
+            const transformedShape = getTransformedShape(shape, rotation);
```

Comment:
Should `getTransformedShape` move to a more generic module, since it's being used on multiple types of annotations, not just regions?